.circleci/config.yml

```diff
@@ -61,6 +65,27 @@ jobs:
             mv googleapis-master googleapis
             export PATH=$PATH:$(pwd)/protobuf/bin
             GOOGLEAPIS=googleapis OUT=$GOPATH/src ./test.sh
+  bazel-test:
+    environment:
+      BAZEL_VERSION: 1.2.1
```

Comment:
Please target bazel version 2+ (it is the newest verson, googleapis CI is build using this bazel version)

Reply:
Done!